### PR TITLE
Refactored Nav with PrevButton, Updated Footer margin and reordered props

### PR DESCRIPTION
**Pull request by robinpiets**

### CODE REVIEW

File: apps/client/src/components/Location/LocationResult.js [hunk 1]
```diff
@@ -62,13 +62,14 @@ const LocationResult = ({
 
       {isActive("locationResult") && (
         <Nav
+          formEnds={!hasSTTR}
+          nextText={!hasSTTR ? "Naar het omgevingsloket" : "Naar de vragen"}
```
Comment (afjlambert) [inline apps/client/src/components/Location/LocationResult.js]: Can you remove the ! and change the order of the ternary options?

**Review by afjlambert — APPROVED** [review]

File: apps/client/src/components/Nav.js [hunk 2]
```diff
@@ -52,54 +54,47 @@ const Nav = ({
     if (onGoToPrev) onGoToPrev(e);
   };
 
+  // <NavStyle {...{ noMarginBottom }}>
```
Comment (svenjens) [inline apps/client/src/components/Nav.js]: Remove comment?

File: apps/client/src/components/Nav.js [hunk 3]
```diff
@@ -5,21 +5,23 @@ import PropTypes from "prop-types";
...
   nextText,
+  noMarginBottom,
   onGoToNext,
   onGoToPrev,
   prevText,
   showNext,
   showPrev,
-  style,
 }) => {
+  console.log(noMarginBottom);
```
Comment (svenjens) [inline apps/client/src/components/Nav.js]: Remove console log

**Review by svenjens — CHANGES_REQUESTED** [review]

File: apps/client/src/components/Location/LocationResult.js (hunk 1 above)
Comment (robinpiets) [reply apps/client/src/components/Location/LocationResult.js]: Better!

File: apps/client/src/components/Nav.js (hunk 2 above)
Comment (robinpiets) [reply apps/client/src/components/Nav.js]: Oops

File: apps/client/src/components/Nav.js (hunk 3 above)
Comment (robinpiets) [reply apps/client/src/components/Nav.js]: Oops 2

**Review by svenjens — APPROVED** [review]

**Pull request merged**